app/models/v2/test_result.rb

```diff
@@ -125,5 +125,9 @@ def self.os_versions
 
       distinct.reorder(major.left, minor.left).reselect(concat, major, minor).map(&:os_version)
     end
+
+    def self.security_guide_versions
+      reselect(security_guide: [:version]).distinct.reorder(:version).map(&:version)
```

Comment:
This sorting of versions is not properly done, you probably need the AREL idiom as [here](https://github.com/RedHatInsights/compliance-backend/blob/master/app/models/v2/security_guide.rb#L38).